index.html

```diff
@@ -1569,10 +1569,24 @@
           Messaging between the local and remote playback devices
         </h3>
         <p>
-          This spec will not mandate communication protocols between the
-          <a>local playback device</a> and the <a>remote playback device</a>,
-          but it should set some guarantees of message confidentiality and
-          authenticity between them.
+          This specification will not mandate communication protocols between
+          the <a>local playback device</a> and the <a>remote playback
+          device</a>, but it should set some guarantees of message
+          confidentiality and authenticity between them.
+        </p>
+      </section>
+      <section>
+        <h3>
+          Secure Contexts
+        </h3>
+        <p>
+          The Remote Playback API is not limited to [[SECURE-CONTEXTS]] because
+          it is meant to provide to a website the ability that user agents
+          native controls usually offer on all media regardless of the
+          <a>browsing context</a>. A user agent MAY limit the API to
```

Comment:
s/MAY/can

The Security and Privacy considerations section is non normative, so it should not contain normative statements (actually, I now see that there is a SHOULD NOT in 7.1 that should be turned into an informative statement as well)

Reply:
👍